app/ports/management/commands/update-portinfo.py

```diff
@@ -0,0 +1,57 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from parsing_scripts import git_update
+from ports.models import Port, LastPortIndexUpdate
+
+
+class Command(BaseCommand):
+    help = "Populates the database with Initial data from portindex.json file"
+
+    def add_arguments(self, parser):
+        parser.add_argument('new',
+                            nargs='?',
+                            default=False,
+                            help="Define a commit till which the update should be processed")
+        parser.add_argument('old',
+                            nargs='?',
+                            default=False,
+                            help="Not recommended. Helps you provide a commit from which update should start")
+
+    def handle(self, *args, **options):
+        # Fetch from rsync
+        Port.RsyncHandler().sync()
+
+        # Open the file
+        data = Port.RsyncHandler().open_file()
+
+        # If no argument is provided, use the commit-hash from JSON file:
+        if options['new'] is False:
```

Comment:
What if we call the option `new_commit` instead of `new`, and use `None` instead of `False`?